### DISCUSSION TITLE
Audio ducking?

### DISCUSSION THREAD

**Discussion by Weroxig** [wwmm/easyeffects#3174] [Ideas]
A way to make all other audio outputs to reduce volume if a program is playing audio.

for me, it's useful because I sometimes have trouble hearing people whilst on calls and I also have other audio on.

**Comment by wwmm** [comment]
The closest thing to what you want would be to set the compressor sidechain input to your microphone device
![image](https://github.com/wwmm/easyeffects/assets/16225172/050215dc-1970-4fb2-899e-5a44ed137245)

This compressor plugin would be in the soundcard pipeline and not in the microphone pipeline. If you set enough compression the sound level of the players audio will be reduced whenever your microphone captures audio.

It isn't possible at the moment to set the output of an arbitrary player as input reference. But the procedure above should help.
